src/main/java/org/red5/server/net/rtmp/RTMPConnection.java

```diff
@@ -122,19 +125,19 @@
 	 * 
 	 * @see org.red5.server.net.rtmp.Channel
 	 */
-	private transient ConcurrentMap<Integer, Channel> channels = new ConcurrentHashMap<Integer, Channel>(3, 0.9f, 1);
+	private transient ConcurrentMap<Integer, Channel> channels = new ConcurrentHashMap<Integer, Channel>(8, 0.9f, 4);
 

```

Comment:
Is this mod working well under load? Seems that these parameters may need to be externalized to the configs since most connections will only have ~3 channels, if I'm not mistaken.